### DISCUSSION TITLE
3 New Shapes! BG The Time Lord has a Corpse

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#323] [Show and tell]
In BG, the Time Lord's single-frame body shape 617 has a "Body" listed at shape 414. Both appear to be recolors of a "sage" shape.

![image](https://user-images.githubusercontent.com/117867844/225805725-98b48662-a979-4a6f-896a-302dc4e4565b.png)

Oddly enough, the Time Lord's corpse is a recolor of the female "sage" body rather than the male version. It has a different blood pattern.

![image](https://user-images.githubusercontent.com/117867844/225806687-7a3ef69b-d637-4776-8224-71e4d2a95aed.png)

I wonder if they did the full recolor of the male "sage" and then just cut the unused frames?

**Comment by KnightCaptainU7** [comment]
I'm going to "backport" the colors from the "sage" to The Time Lord so he has all frames available.

Here's a view of the palette by relative darkness.
![image](https://user-images.githubusercontent.com/117867844/225930050-59bf404a-4269-49e9-b3a3-d8ae4a9f4c09.png)

**Comment by KnightCaptainU7** [comment]
In trying to do all of this I hit multiple bugs in other projects.
GIMP 2.10 has a broken color palette selector, so when you color-grab a pixel it no longer shows the same on the indexed map. I had to revert to 2.8.22. It's a minor feature, they say, so it may not be fixed.
Then in Notepad++, I was not removing tabs correctly, and I only found that out after ImageMagick CLI kept growling errors back at me.

Here's my first attempt, which is not as papal-white as I had hoped:
![image](https://user-images.githubusercontent.com/117867844/226025662-75a64035-775c-42da-a1c2-1a87c1c0f77d.png)

**Comment by KnightCaptainU7** [comment]
The colors were swapped just one-for-one by darkness level. I think the shape looks more like an Undead Monk, so here it is available for download:
https://github.com/KnightCaptainU7/exult-art/tree/main/graphics/shapes

**Comment by DominusExult** [comment]
Bah, with each version of Gimp it seems they are losing features ;(

**Comment by KnightCaptainU7** [comment]
Okay, I had to do manual checks, pixel by pixel to get this:
![image](https://user-images.githubusercontent.com/117867844/226077356-2a005ca5-92a0-4d30-8eea-1e1deb4fc85d.png)
But now that I know the canonical color swaps, I can apply them to the rest of the sage frames.

**Reply by KnightCaptainU7** [reply]
The magic formula:
"C:\Program Files\ImageMagick-7.0.8-Q16\magick.exe" convert BG_Sage_0318_%02d.png[0-33] -fill #ecd8c4 -opaque #cc987c -fill #e0c0b0 -opaque #bc805c -fill #ccb4a0 -opaque #ac643c -fill #bc9c94 -opaque #9c5024 -fill #9c8474 -opaque #8c3c0c -fill #7c6458 -opaque #7c2c00 -fill #504038 -opaque #6c2400 -fill #604844 -opaque #602000 -fill #342c24 -opaque #501c00 -fill #241818 -opaque #441400 -fill #342c24 -opaque #341000 -fill #ecd8c4 -opaque #ac7c64 -fill #ccb4a0 -opaque #986c58 -fill #ac9080 -opaque #805c4c -fill #8c7464 -opaque #6c4c3c -fill #6c544c -opaque #543c30 -fill #504038 -opaque #3c2c24 -fill #fcfcfc -opaque #c48c74 -define png:preserve-colormap=true New_Time_Lord_Canon_14_%02d.png[0-33]

**Comment by KnightCaptainU7** [comment]
Done!
![image](https://user-images.githubusercontent.com/117867844/226081116-edc351ff-8275-4ea7-8c30-017b617aa181.png)
I re-added the flesh-colored hands since they were also overridden by the script above.

**Reply by KnightCaptainU7** [reply]
Available here: https://github.com/KnightCaptainU7/exult-art/tree/main/graphics/shapes

**Comment by KnightCaptainU7** [comment]
I liked one of the other recolors, so here is the Gray Liche:
![image](https://user-images.githubusercontent.com/117867844/226148298-3ce53dfe-98fa-48d1-b5f2-7963f18a6236.png)

Sorry @DominusExult but still no Team Instinct Liche.
